core/src/test/java/io/questdb/test/griffin/CreateTableDedupTest.java

```diff
@@ -162,6 +162,28 @@ public void testCreateTableSetTypeSqlSyntaxErrors() throws Exception {
         });
     }
 
+    @Test
+    public void testCreateTableWithDoubleQuotes() throws Exception {
+        String tableName = "ABB_Events2";
+        assertMemoryLeak(ff, () -> {
+            ddl(
+                    "CREATE TABLE '" + tableName + "' (\n" +
+                            "  Status SYMBOL capacity 16 CACHE,\n" +
+                            "  Reportedtime TIMESTAMP\n" +
```

Comment:
Could you also test with quoted identifier that contains a whitespace character ?

Reply:
4175bcf16000896346840a4bc5907d020c44a1e7